src/ARControllerNFT.ts

```diff
@@ -193,11 +193,13 @@ export default class ARControllerNFT implements AbstractARControllerNFT {
     let k, o;
 
     // get NFT markers
-    for (k in this.nftMarkers) {
+    /*for (k in this.nftMarkers) {
```

Comment:
It seems that with latest changes this code is not required. The NFT marker is detected and trackied with sinlge and multi mode.